airflow/models/pool.py

```diff
@@ -161,21 +163,26 @@ def slots_stats(
         from airflow.models.taskinstance import TaskInstance  # Avoid circular import
 
         pools: dict[str, PoolStats] = {}
+        pool_includes_deferred: dict[str, bool] = {}
 
-        query = select(Pool.pool, Pool.slots)
+        query = select(Pool.pool, Pool.slots, Pool.include_deferred)
 
         if lock_rows:
             query = with_row_locks(query, session=session, **nowait(session))
 
         pool_rows = session.execute(query)
-        for (pool_name, total_slots) in pool_rows:
+        for (pool_name, total_slots, includes_deferred) in pool_rows:
```

Comment:
Please unify whether you want to use _include_ or _includes_.

Reply:
Done